mne_icalabel/iclabel/tests/test_label_components.py

```diff
@@ -1,11 +1,12 @@
 import pytest
+from pathlib import Path
 from mne.datasets import sample
 from mne.io import read_raw
 from mne.preprocessing import ICA
 
 from mne_icalabel.iclabel import iclabel_label_components
 
-directory = sample.data_path() / "MEG" / "sample"
+directory = Path(sample.data_path()) / "MEG" / "sample"
```

Comment:
`.data_path()` should already return Path

Reply:
Ah yes. you're right. this was only needed for v0.24-

Reply:
I think you forgot to revert this / change it according to @mscheltienne's proposal

Reply:
I did! The diff is outdated here :p.

Reply:
Sorry, my bad!